src/index.tsx

```diff
@@ -45,9 +56,12 @@ export type OCRFrame = {
 /**
  * Scans OCR.
  */
+const plugin = VisionCameraProxy.getFrameProcessorPlugin('scanOCR');
```

Comment:
```suggestion
const plugin = VisionCameraProxy.initFrameProcessorPlugin('scanOCR');
```

I just renamed this in VisionCamera 3.5.0. Init is more accurate than get.

Reply:
Done!